### PR TITLE
feat(ssr-catalogue): add 'browse' to landing page buttons so they are…

### PR DESCRIPTION
**Pull request by mswertz**
… more actionable

### CODE REVIEW

**Review by connoratrug — COMMENTED** [review]
The code works but would advise against this change, if you do want the multi word button ( which makes the selection harden ) if would suggest using sentence case for the button title

File: apps/nuxt3-ssr/components/landing/CardPrimary.vue [hunk 1]
```diff
@@ -31,7 +31,7 @@ defineProps<{
       {{ description }}
     </p>
     <NuxtLink :to="link">
-      <Button :label="title" type="primary" size="medium" />
+      <Button :label="'Browse '+title" type="primary" size="medium" />
```
[the remark this answers is not in the document]
Comment (mswertz) [reply apps/nuxt3-ssr/components/landing/CardPrimary.vue:34]: what do you propose? to lowercase it?

**Comment by mswertz** [comment]
> The code works but would advise against this change, if you do want the multi word button ( which makes the selection harden ) if would suggest using sentence case for the button title

Okay, good suggestion. Can you do that for me?

**Comment by mswertz** [comment]
Ah, that was simpler than I thought, thx

**Comment by connoratrug** [comment]
> Ah, that was simpler than I thought, thx

well its a lot more complex then i thought, apparently the font-display sets the font to Bebas Neue that only seams to be included in uppercase variant ( the uppercase class seams redundant, at least for this theme). 

Suggest you decide if you want to merge with the all UPPERCASE variant for now

**Comment by mswertz** [comment]
okay, I propose to merge this now to appease our stakeholders and then start taking a step back looking forward to the network catalogue

**Pull request merged**